src/main/python/apache/aurora/executor/bin/thermos_executor_main.py

```diff
@@ -183,6 +183,14 @@
     help='Maximum number of rotated stdout/stderr logs emitted by the thermos runner.')
 
 
+app.add_option(
+    '--disk-log-level',
+    dest='disk_log_level',
+    type=str,
+    default='INFO',
+    help='Disk Log Level DEBUG/INFO.')
```

Comment:
What about also specifying the other log levels in the help? DEBUG, INFO, WARN, FATAL, ERROR